Cargo.toml

```diff
@@ -30,7 +30,7 @@ either = { version = "1.1.0" }
 debugtrace = { version = "0.1.0" }
 tendril = { version = "0.4.3", optional = true }
 
-compiletest_rs = "0.7.1"
+compiletest_rs = { version = "0.7.1", optional = true }
```

Comment:
Shouldn't this be the latest version? I am getting compile errors with this older version:
Latest version: https://crates.io/crates/compiletest_rs